EVENT.md

```diff
@@ -980,6 +980,10 @@ were reported.
 
 In order to achieve the privacy goals listed above the API has various rate limits, which can be found [here](https://github.com/WICG/attribution-reporting-api/blob/main/params/chromium-params.md) for Chromium.
 
+Note: Browsers may choose to allow separate rate limits for the embedded site 
```

Comment:
Is `embedded site` standard terminology? If so, let's link to a definition of it. Otherwise, let's explain further. Same for the spec changes.

Reply:
Also, IIUC the spec changes aren't limited to rate limits. Rather, they effectively replace the context origin in *all uses* across the entire spec.

For sources, this includes ["max pending sources per source origin"](https://wicg.github.io/attribution-reporting-api/#max-pending-sources-per-source-origin) and ["max destinations covered by unexpired sources"](https://wicg.github.io/attribution-reporting-api/#max-destinations-covered-by-unexpired-sources), which aren't explicitly considered "rate limits."

It also includes [the source site in verbose debug report bodies](https://wicg.github.io/attribution-reporting-api/#ref-for-attribution-source-source-site%E2%91%A0%E2%91%A2).

For aggregatable reports, it affects the [effective attribution destination](https://wicg.github.io/attribution-reporting-api/#aggregatable-report-effective-attribution-destination) and therefore ["max aggregatable attribution reports per attribution destination"](https://wicg.github.io/attribution-reporting-api/#max-aggregatable-attribution-reports-per-attribution-destination).

IMO we should make it explicit that the context origin is entirely replaced, not just for rate-limit purposes.

Reply:
I used “rate limits” to cover those limits as well. The destination limit is also discussed in this section, so I think it’s fine. I can mention storage limit explicitly if you feel strongly.

Reply:
The aforementioned limits aren't rate limits, as they do not expire after a specific window, and they are mentioned in different parts of the spec and explainer.

Reply:
Updated to mention storage limit, rate-limits, and destination limits explicitly.